templates/payg/bigquery.mdx

```diff
@@ -149,17 +149,17 @@ In the plan setting, we only need to define a monthly billing period and set a s
   </Tab>
 </Tabs>
 
-## Step 2: Create a wallet to offer free credits
-_“Start your 90-day free BigQuery trial which includes $300 worth of free usage”_
+## Step 2- Create a wallet to offer free credits
+"Start your 90-day free BigQuery trial which includes $300 worth of free usage"
```

Comment:
Is it expected that the underscores (`_`) will be removed as well here?

Reply:
> Is it expected that the underscores (`_`) will be removed as well here?

I don't think they were useful tbh